rotkehlchen/tests/api/test_users.py

```diff
@@ -309,8 +309,8 @@ def test_user_creation_with_invalid_premium_credentials(
         'premium_api_secret': VALID_PREMIUM_SECRET,
     }
     for side_effect, expected_msg in (
-        (RemoteError('Invalid API-KEY'), 'Invalid API-KEY'),
-        (PremiumPermissionError('Device limit of 4 exceeded'), 'Device limit of 4 exceeded'),
+        (RemoteError('Invalid API-KEY'), 'Invalid API-KEY.'),
+        (PremiumPermissionError('Device limit of 4 exceeded'), 'Device limit of 4 exceeded. _DEVICE_LIMIT_LINK_'),  # noqa: E501
```

Comment:
Since you have underscode already in the text maybe do

```suggestion
        (PremiumPermissionError('Device limit of 4 exceeded'), 'Device limit of 4 exceeded. :DEVICE_LIMIT_LINK:'),  # noqa: E501
```

Also why not have the link in the backend instead of the frontend? Is it dinamic based on the tier?